src/popup/popup.js

```
@@ -330,17 +330,35 @@ function sanitizeFilename(filename) {
+        'html': 'html',
+        'json': 'json'
+    };
+    
+    // Update file type if we can determine it from the extension
+    if (fileExtension && fileTypeMap[fileExtension]) {
+        fileType = fileTypeMap[fileExtension];
     }
-
+    
+    // Create folder structure based on file type
+    let fileTypeFolder = sanitizeFilename(fileType) + '/';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)